src/globals/helpers/_media-queries.scss

```diff
@@ -1,8 +1,8 @@
 // We use sass-mq module for media queries
 
 // This is a horrible, horrible hack to prevent the 'dev mode' CSS to display
-// the current breakpoint from being included multiple times. 
-// 
+// the current breakpoint from being included multiple times.
+//
 // We can't use the `exports` mixin for this because import directives cannot be
```

Comment:
Non-blocking but if we vendor this is this hack necessary any more? @36degrees

Reply:
No, not if we wrap the whole of sass-mq in an `exports`.